va/va.go

```diff
@@ -781,6 +783,14 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, req *v
 	validationLatency := time.Since(vStart)
 	logEvent.ValidationLatency = validationLatency.Round(time.Millisecond).Seconds()
 
+	// TODO(#7321): Remove this when we have collected enough data.
+	for _, record := range records {
+		if record.UsedRSAKEX {
+			logEvent.UsedRSAKEX = true
+			break
+		}
+	}
```

Comment:
This is a little misleading because it suggests that intermediate steps in a redirect chain could have this set to true. Options:

 - Change the comment to caveat "only the last one will ever be set"
 - Make that explicit by only checking the last one
 - Set the bool inside processRedirect, and keep this loop the same